servers/src/main/java/tachyon/worker/block/allocator/AllocatorFactory.java

```diff
@@ -35,8 +35,10 @@ public static Allocator create(AllocatorType allocatorType, BlockMetadataManager
         return new GreedyAllocator(metaManager);
       case MAX_FREE:
         return new MaxFreeAllocator(metaManager);
+      case ROUND_ROBIN:
+        return new RoundRobinAllocator(metaManager);
       default:
-        return new GreedyAllocator(metaManager);
+        return new MaxFreeAllocator(metaManager);
```

Comment:
I didn't change the default.

I think the rebased_worker_wip branch uses MaxFreeAllocator, while the
master branch uses GreedyAllocator.

Let me know if I'm wrong. I can definitely change back to Greedy.

~t

On 8 July 2015 at 12:21, Calvin Jia <notifications@github.com> wrote:

> In
> servers/src/main/java/tachyon/worker/block/allocator/AllocatorFactory.java
> <https://github.com/amplab/tachyon/pull/1040#discussion_r34187655>:
>
> >        default:
> > -        return new GreedyAllocator(metaManager);
> > +        return new MaxFreeAllocator(metaManager);
>
> No need to change the default in this PR?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amplab/tachyon/pull/1040/files#r34187655>.
>



-- 
Tianyin XU,
http://cseweb.ucsd.edu/~tixu/